### PR TITLE
Added an appdata.xml file for Linux software gallery integration

### PR DESCRIPTION
**Pull request by Mailaender**
https://people.freedesktop.org/~hughsient/appdata/ gives some background and https://software.opensuse.org/appstore is a web frontend to show you how it could look like.

Note: the screenshot is temporarily. I didn't find a suitable non-animated royality-free overview screenshot. You may want to tweak this yourself.

### CODE REVIEW

File: desktop/aseprite.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">aseprite.desktop</id>  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
```
Comment (dacap) [inline desktop/aseprite.appdata.xml]: Hi @Mailaender, actually Aseprite is distributed under GPL-2.0, I'm thinking to change it to GPL-3.0 but at this moment it's 2.0.

File: desktop/aseprite.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <name>Aseprite</name>
+  <summary>Animated sprite editor and pixel art tool</summary>
+  <description>
+    <p>Aseprite lets you create 2D animations for videogames. From sprites, to pixel-art, retro style graphics, and whatever you like about the 8-bit (and 16-bit) era.</p>
+  </description>
+  <url type="homepage">http://www.aseprite.org/</url>
+  <screenshots>
+    <screenshot>http://i.imgur.com/1myue2n.gif</screenshot>
```
Comment (dacap) [inline desktop/aseprite.appdata.xml]: We might include other screenshot for this store.

File: desktop/aseprite.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">aseprite.desktop</id>  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>Aseprite</name>
+  <summary>Animated sprite editor and pixel art tool</summary>
+  <description>
+    <p>Aseprite lets you create 2D animations for videogames. From sprites, to pixel-art, retro style graphics, and whatever you like about the 8-bit (and 16-bit) era.</p>
```
Comment (dacap) [inline desktop/aseprite.appdata.xml]: We could add a paragraph saying something like "You can contribute to the official development buying a copy of Aseprite from its website."

File: desktop/aseprite.appdata.xml (hunk 2 above)
Comment (Mailaender) [inline desktop/aseprite.appdata.xml]: Something like http://davidcapello.com/images/aseprite.png without the Windows 10 border would be great.

**Comment by Mailaender** [comment]
Updated this a bit.

**Comment by dacap** [comment]
Sorry for this back and forth @Mailaender, this weekend I've decided to remove GPL license with a the [EULA for Aseprite](https://github.com/aseprite/aseprite/blob/master/EULA.txt) which is incompatible with Linux distributions. Soon I'll publish my thoughts about this.

**Comment by dacap** [comment]
I'll investigate the possibility to distribute an AppImage so we can release one package for all distributions.

**Comment by probonopd** [comment]
> I'll investigate the possibility to distribute an AppImage so we can release one package for all distributions.

Check out https://github.com/probonopd/AppImages/blob/master/recipes/aseprite/Recipe as a starting point. This uses the deb provided on the official download page and converts it into an AppImage.

The experimental AppImage can be downloaded from https://bintray.com/probono/AppImages/Aseprite#files but most likely some additional fine-tuning is required, ideally by the Aseprite upstream development team.

**Comment by Mailaender** [comment]
You are aware that those binaries are in direct violations of @dacap's https://github.com/aseprite/aseprite/blob/master/EULA.txt?

**Comment by probonopd** [comment]
No.

> Evaluation versions available for download from David Capello's websites may be freely distributed.

**Comment by Mailaender** [comment]
Ah, I see.

**Comment by dacap** [comment]
@probonopd thanks for the help with this one, I'll take a look at it on Monday and see if I can start packaging it as an AppImage next week.

**Comment by probonopd** [comment]
> Thanks packaging this for every Linux distribution instead of just Ubuntu.

My recipe uses Ubuntu packages as input, but the output AppImage runs on almost every Linux distribution, not just Ubuntu.

**Pull request closed without merging**